cpp/ql/src/Telemetry/CompilerErrors.ql

```diff
@@ -0,0 +1,12 @@
+/**
+ * @name Compiler errors
+ * @description A count of all compiler errors, grouped by error text.
+ * @kind metric
+ * @tags summary telemetry
+ * @id cpp/telemetry/compiler-errors
+ */
+
+import Metrics
+
+from CppMetrics::ErrorCount m
+select m.toString(), m.getValue()
```

Comment:
I'm a bit concerned that this might completely blow up on real-world projects to such an extent that the SARIF will be rejected when uploaded. Especially because the error messages can in principle be several lines long.